store/postgres/src/filter.rs

```diff
@@ -271,9 +271,72 @@ fn store_filter_by_mode<'a>(
         }
         // Is `attribute` equal to some `v` in `query_values`?
         StoreFilter::In(attribute, query_values) => {
-            query_values.into_iter().try_fold(query, |q, v| {
-                store_filter_by_mode(q, StoreFilter::Equal(attribute.clone(), v), FilterMode::Or)
-            })?
+            let op = " = ANY (";
```

Comment:
Diesel query builder has support for this https://docs.diesel.rs/diesel/expression_methods/trait.ExpressionMethods.html#method.eq_any, I wonder if we can use it here.

Reply:
I don't see a reasonable way through using diesel expression methods like `eq_any()` with our`jsonb` attributes right now.  

We could create a custom type that uses diesel-dynamic-schema to build the structs at runtime, but I think that would also require looking at the schema to see what attributes are on each entity.